rasa/core/policies/ensemble.py

```diff
@@ -275,6 +276,9 @@ def _best_policy_prediction(
             for prediction in predictions:
                 prediction.probabilities[index_of_rejected_action] = 0.0
 
+        predictions = plugin_manager().hook.filter_and_rerank_actions(
```

Comment:
To use only the first result, you can mark the hook with `@hookspec(firstresult=True)` 
https://pluggy.readthedocs.io/en/stable/#first-result-only

Reply:
Also we should cover the case when people are not using rasa pro thus the hook returning `None` (I believe). In this case the original predictions should not be overwritten